packages/hop-node/src/watchers/BondTransferRootWatcher.ts

```diff
@@ -44,90 +37,56 @@ class BondTransferRootWatcher extends BaseWatcher {
   }
 
   async pollHandler () {
+    if (!this.isL1) {
```

Comment:
Since this watcher only requires L1 and doesn't use sibling watchers, does it make to sense to only initialize the L1 watcher and not the others in watchers.ts?

Reply:
I think that makes a lot of sense. It looks like we have a similar setup for these watchers as well—would you imagine these watcher initializations are all handled in `watcher.ts` as well?

![Screen Shot 2021-08-18 at 8 40 30 AM](https://user-images.githubusercontent.com/9441295/129928579-a5aa157e-d980-4a8e-a024-9e2a3b3dbddc.png)